ci/aws-analytical-env/jobs/mirror-git-repo-s3.yml

```diff
@@ -0,0 +1,19 @@
+jobs:
+  - name: mirror-git-repo-s3
+    plan:
+      - get: chs-uc-lab-repo
```

Comment:
I think you want this to trigger the job and `trigger` defaults to false, so I think you need to set it to true

Reply:
Amended code